payments-core/src/main/java/com/stripe/android/networking/payments/PaymentIntentFlowResultProcessor.kt

```diff
@@ -0,0 +1,75 @@
+package com.stripe.android.payments
+
+import android.content.Context
+import com.stripe.android.Logger
+import com.stripe.android.PaymentIntentResult
+import com.stripe.android.model.PaymentIntent
+import com.stripe.android.networking.ApiRequest
+import com.stripe.android.networking.StripeRepository
+import kotlinx.coroutines.withContext
+import kotlin.coroutines.CoroutineContext
+
+/**
+ * Processes the result of a [PaymentIntent] confirmation.
+ */
+internal class PaymentIntentFlowResultProcessor(
```

Comment:
this class is almost identical to `SetupIntentFlowResultProcessor`, can we extract the common logic into `PaymentFlowResultProcessor`?